utils/obsproc/Ghrsst2Ioda.h

```diff
@@ -157,21 +157,33 @@ namespace gdasapp {
       std::vector<std::vector<float>> lon2d_s;
       std::vector<std::vector<float>> lat2d_s;
       std::vector<std::vector<float>> obserror_s;
-      std::vector<std::vector<float>> seconds_s;
       if ( fullConfig_.has("binning") ) {
         sst_s = gdasapp::superobutils::subsample2D(sst, mask, fullConfig_);
         lon2d_s = gdasapp::superobutils::subsample2D(lon2d, mask, fullConfig_);
         lat2d_s = gdasapp::superobutils::subsample2D(lat2d, mask, fullConfig_);
         obserror_s = gdasapp::superobutils::subsample2D(obserror, mask, fullConfig_);
-        seconds_s = gdasapp::superobutils::subsample2D(seconds, mask, fullConfig_);
       } else {
         sst_s = sst;
         lon2d_s = lon2d;
         lat2d_s = lat2d;
         obserror_s = obserror;
-        seconds_s = seconds;
       }
 
+      // Non-Superobing part only applied to datetime
```

Comment:
It's not a problem of efficiency, it 's a bug that we haven't been able to figure out yet. Re-write your comment and todo to reflect this. We WILL forget what we did a few month down the road when we decide this needs to be fixed properly!